utils/docker-compose/scripts/generate.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
```

Comment:
This scripts uses bash'ism => s:/bin/sh:/bin/bash:

Reply:
fixed